docs/tutorials/word-tutorial.md

```diff
@@ -285,56 +301,44 @@ In this step of the tutorial, you'll apply a built-in style to text, apply a cus
 
 ### Test the add-in
 
-1. In the Git bash window, or Node.JS-enabled system prompt, from the previous stage tutorial is still open, enter Ctrl+C twice to stop the running web server. Otherwise, open a Git bash window, or Node.JS-enabled system prompt, and navigate to the **Start** folder of the project.
-
-     > [!NOTE]
-     > Although the browser-sync server reloads your add-in in the task pane every time you make a change to any file, including the app.js file, it does not retranspile the JavaScript, so you must repeat the build command in order for your changes to app.js to take effect. In order to do this, you need to kill the server process so that the prompt appears and you can enter the build command. After the build, you restart the server. The next few steps carry out this process.
+1. [!include[Start server and sideload add-in instructions](../includes/tutorial-word-start-server.md)]
 
-2. Run the command `npm run build` to transpile your ES6 source code to an earlier version of JavaScript that is supported by all the hosts where Office Add-ins can run.
+2. If the add-in task pane isn't already open in Word, go to the **Home** tab and choose the **Show Taskpane** button in the ribbon to open it.
 
-3. Run the command `npm start` to start a web server running on localhost.   
+3. Be sure there are at least three paragraphs in the document. You can choose the **Insert Paragraph** button three times. *Check carefully that there's no blank paragraph at the end of the document. If there is, delete it.*
 
-4. Reload the task pane by closing it, and then on the **Home** menu select **Show Taskpane** to reopen the add-in.
+4. In Word, create a custom style named "MyCustomStyle". It can have any formatting that you want.
```

Comment:
Consider linking "custom style" to this support page: https://support.office.com/article/Customize-or-create-new-styles-d38d6e47-f6fc-48eb-a607-1eb120dec563